client/application/api/api.factory.js

```diff
@@ -14,7 +14,9 @@
       login: login,
       logout: logout,
       getRoomList: getRoomList,
-      reserveSlot: reserveSlot
+      reserveSlot: reserveSlot,
```

Comment:
No this is a request factory, basically just a function name, it is not going to be passed to the back end